modules/serverless-rn-1-10-0.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-10-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.10.0
+
+[id="new-features-1-10-0_{context}"]
+== New features
+* {ServerlessProductName} now uses Knative Operator 0.16.0.
+* {ServerlessProductName} now uses Knative Serving 0.16.0.
+* {ServerlessProductName} uses Knative Eventing 0.16.0.
+* {ServerlessProductName} now uses Kourier 0.16.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.16.1.
```

Comment:
I'm in active discussions with upstream regarding their usage of patch versions and how much of a PITA it is to herd the zoo of releases.

Maybe we should opt to not specify down to the patch version here?

Reply:
@markusthoemmes as long as this list is correct I'd include it as is for now, but for next release we can change our approach?